hudi-client/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -739,4 +728,113 @@ public HoodieWriteConfig build() {
       return config;
     }
   }
+
+  // --------------------------------------------------------------------------------
+  // ------------------------  ConfigOption Part  -----------------------------------
+  // --------------------------------------------------------------------------------
+
+  public void setString(ConfigOption<String> option, String value) {
+    Preconditions.checkArgument(option != null);
+    Preconditions.checkArgument(value != null);
```

Comment:
Replace all Preconditions.checkArgument() with ValidationUtils.checkArgument() <--- once the PR for that has been merged

Reply:
Got it, thanks. :)